<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeTableView</class>
 <widget class="QWidget" name="DeTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeTableView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
     border: 1px solid black;
     border-radius:2px;
     min-width: 18px;
     min-height: 18px;
}

QLabel
{
   font:14px &quot;微软雅黑&quot;;
}

QTableView
{
	font: bold 12px &quot;微软雅黑&quot;;    
    border:none;
    background-color: #E9E9E9;
	selection-background-color: #42BDAA;
	selection-color: #FFFFFF;
}

QTableView
{
	gridline-color: #E9E9E9;
}

 QTableView QTableCornerButton::section
 {
     border: 0px outset rgb(209, 209, 209);
     border-bottom-width:1px;
     border-right-width:1px;
 }


QHeaderView::section
{
    border:1px solid white;
	background-color: #C2C2C2;    
	color: rgb(255, 255, 255);    
	border-top-width:0px;
	border-bottom-width:1px;	
	border-right-width:1px;
	border-left-width:0px;
    font:bold 14px &quot;微软雅黑&quot;;
    min-height:32px;
    border-color: #E9E9E9;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableView" name="tableView">
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: #E9E9E9;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>18</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="firstBtn">
        <property name="text">
         <string>|&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="prevBtn">
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="nextBtn">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="lastBtn">
        <property name="text">
         <string>&gt;|</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
